2.4/root/usr/bin/run-mongod

```diff
@@ -47,9 +47,7 @@ function cleanup() {
   fi
   echo "=> Shutting down MongoDB server ..."
   if [ -f "${MONGODB_PID_FILE}" ]; then
-    kill -2 $(cat ${MONGODB_PID_FILE})
-  else
-    pkill -2 mongod
+    mongod --shutdown $mongo_common_args
```

Comment:
IMPOV it would be better to use only pgrep a pkill instead of using || true or set +-e (it also removes dependency on PID file and it is more clear). What do you think about current implementation?